acquisition.py

```diff
@@ -86,7 +91,87 @@ def get_reward_batch(self, inputs_af_base):  # inputs_af = list of ...
         inputs = pad_sequence(inputs_af, batch_first=True, padding_value=0.0)
 
         self.load_best_proxy()
-        self.proxy.model.eval()
+        self.proxy.model.train()
```

Comment:
Do we actually want to add dropout one-shot inference (proxy values without uncertainty ?)

Reply:
to be discussed I guess